files/build_templates/init_cfg.json.j2

```diff
@@ -99,6 +100,20 @@
             "rate_limit_interval" : "600",
             "available_mem_threshold": "10.0"
         }{%if not loop.last %},{% endif -%}
+{% endfor %}
+    },
+    "SYSLOG_CONFIG": {
+        "GLOBAL": {
+            "rate_limit_interval" : "0",
```

Comment:
Just to confirm - you do not want rate limiting on host?

Reply:
Yes, host side has no rate limiting by default.